src/ucp/wireup/select.c

```diff
@@ -1418,8 +1433,7 @@ static double ucp_wireup_aux_score_func(ucp_context_h context,
             iface_attr->overhead + remote_iface_attr->overhead));
 }
 
-ucs_status_t ucp_wireup_select_aux_transport(ucp_ep_h ep,
-                                             const ucp_ep_params_t *params,
+ucs_status_t ucp_wireup_select_aux_transport(ucp_ep_h ep, const ucp_ep_params_t *params,
```

Comment:
what is the motivation?
before that we had every parameter on a new line, which (imo) looks better

Reply:
revert the change